storybookjs/storybook
# Addon-knobs: : add number config options

Fix inconsistency in number API

fix step,min,max attributes in number type

Issue:

## What I did

Fixed inconsistency in number type.

## How to test

when setting the options of a number, the step,min,max attributes now get applied.

Is this testable with jest or storyshots? Yes, but I didn't write a test.

Does this need a new example in the kitchen sink apps? No.

Does this need an update to the documentation? Not really, but I could add an example.

If your answer is yes to any of these, please make sure to include it in your PR.
